packages/react-native/src/theme/__tests__/utils.spec.ts

```diff
@@ -19,13 +17,13 @@ const tokens: Tokens = {
       },
     },
   },
-  components: {},
 };
 
 describe('utils', () => {
   describe('setupTokens', () => {
+    const finalTokens = setupTokens(customTokens);
+
     it('should replace token references with values', () => {
-      const finalTokens = setupTokens(tokens);
```

Comment:
In preparation for more tests that can reuse it

Reply:
FWIW prefer to keep test details isolated to prevent them from bleeding outside of their test. For example, if `finalTokens` was to be mutated in fictitious `testOne`, and fictitious `testTwo` is testing against `finalTokens`, if we update `testOne` it can break `testTwo`